lib/Yam/ControlCenter.pm

```diff
@@ -0,0 +1,374 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles Control Center
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::ControlCenter;
+use parent 'YaST::PageBase';
+use strict;
+use warnings;
+use testapi;    # needed because of workaround_bsc1206966
+
+sub init {
+    my $self = shift;
+    $self->{lbl_yast_cc} = $self->{app}->label({label => 'YaST Control Center'});
+    $self->{slb_groups} = $self->{app}->selectionbox({id => 'groups'});
+    $self->{slb_progs} = $self->{app}->selectionbox({id => 'progs'});
+    $self->{btn_help} = $self->{app}->button({id => 'help'});
+    $self->{btn_run} = $self->{app}->button({id => 'run'});
+    $self->{btn_quit} = $self->{app}->button({id => 'quit'});
+    return $self;
+}
+
+sub get_launcher_page {
+    my ($self) = @_;
+    die 'YaST Control Center is not displayed' unless $self->{lbl_yast_cc}->exist();
+    return $self;
+}
+
+sub workaround_bsc1206966 {
+    my ($self) = @_;
+    my $currentgroup = $self->{slb_groups}->selected_items();
+    send_key 'shift-tab';
+    if ($currentgroup eq 'Software') {
+        send_key 'down';
+        send_key 'up';
+    }
+    else {
+        send_key 'up';
+        send_key 'down';
+    }
+}
+
+# private methods collection
+
+sub _select_software {
+    my ($self) = @_;
+    $self->{slb_groups}->select('Software');
+    workaround_bsc1206966();
+}
+
+sub _select_online_update {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Online Update');
+}
+
+sub _select_software_management {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Software Management');
+}
+
+sub _select_add_on_products {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Add-On Products');
+}
+
+sub _select_media_check {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Media Check');
+}
+
+sub _select_software_repositories {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Software Repositories');
+}
+
+
```

Comment:
extra line, let's not do artificial separation.
I also would remove the private and public lines of comments, I don't think they help much, but without comments the separation between methods should be the same.
It is clear by the workaround added there that there is another section.